setup/helm/varadhi/values/controller_values.yaml

```diff
@@ -0,0 +1,240 @@
+      name: datalog
+      size: 10Mi
+    conf:
+      name: zk-conf
+    scripts:
+      name: zk-scripts
+  service:
+    annotations:
+  configMapName: zookeeper
+  scriptsConfigMapName: zookeeper-scripts
+  logLevel: INFO
+  configData:
+    tickTime: 2000
+    globalOutstandingLimit: 5000
+    clientPort: 2181
+    maxClientCnxns: 10000
+    initLimit: 150
+    syncLimit: 15
+    skipACL: yes
+    purgeInterval: 24
+    snapRetainCount: 10
+    peerType: "participant"
+
+images:
+  pullPolicy: IfNotPresent
+  appImage:
+    registry: jfrog.fkinternal.com
+    repository: varadhi-oss/varadhi
+    tag: local-c1-remote-ip
+  zookeeper:
+    registry: docker.io
+    repository: zookeeper
+    tag: 3.7.0
+
+
+#############################################################################
+#  Config Settings for respective Varadhi components and their dependencies
+############################################################################
+
+####################################################
+# Varadhi app -- Server (rest api + producer) config
+####################################################
+varadhi:
+  app:
+    name: controller
+    configMapName: controller
+    restOptions:
+      deployedRegion: "default"
+      defaultOrg: "default"
+      defaultTeam: "public"
+      defaultProject: "public"
+    producerOptions:
+      producerCacheBuilderSpec: ""
+      topicCacheBuilderSpec: ""
+      metricEnabled: true
+    vertxOptions:
+      eventLoopPoolSize: 1
+      workerPoolSize: 1
+      internalBlockingPoolSize: 1
+    verticleDeploymentOptions:
+      instances: 1
+      worker: false
+      workerPoolSize: 2
+    deliveryOptions:
+      timeoutMs: 1000
+      tracingPolicy: "PROPAGATE"
+    httpServerOptions:
+      port: 18488
+      alpnVersions: [ "HTTP_1_1", "HTTP_2" ]
+      decompressionSupported: false
+      useAlpn: true
+    authenticationEnabled: false
+    authentication:
+      jwt:
+        jwksUrl: "http://localhost:9090/default/jwks"
+        options:
+          issuer: "http://localhost:9090/default"
+          ignoreExpiration: true
+    authorizationEnabled: true
+    authorization:
+      superUsers: [ "thanos" ]
+      providerClassName: "com.flipkart.varadhi.auth.DefaultAuthorizationProvider"
+    messagingStackOptions:
+      providerClassName: "com.flipkart.varadhi.pulsar.PulsarStackProvider"
+    metaStoreOptions:
+      providerClassName: "com.flipkart.varadhi.db.ZookeeperProvider"
+    featureFlags:
+      leanDeployment: false
+    member:
+      roles: [ "Controller" ]
+      cpuCount: 1
+      nicMBps: 100
+
+#############################################
+# Varadhi Server logging config
+#############################################
+logging:
+  configMapName: logging
+  level: info
+
+#############################################
+# Metastore config -- zookeeper
+#############################################
+metastore:
+  configMapName: metastore
+  zookeeper:
+    connectUrl: "127.0.0.1:2181"
```

Comment:
in server value file this is set as 
varadhi-zookeeper-0.varadhi-zookeeper.default.svc.cluster.local. why?